spikeinterface/postprocessing/unit_localization.py

```diff
@@ -285,7 +284,7 @@ def compute_monopolar_triangulation(waveform_extractor, optimizer='minimize_with
     return unit_location
 
 
-def compute_center_of_mass(waveform_extractor, peak_sign='neg', num_channels=10):
+def compute_center_of_mass(waveform_extractor, peak_sign='neg', radius_um=50):
```

Comment:
excelent!

Reply:
do we have the same parameters for pike localization ?

Reply:
I'll check

Reply:
The method_kwargs propagation is a bit obscure in `spike_localization`, since it calls the `localize_peaks`. In that case the arg should be `local_radius_um`. We have to refactor the classes so that we can expose the method kwargs automatically

Reply:
ok